src/Aws/CloudFront/Core.hs

```diff
@@ -56,7 +56,18 @@ instance Monoid.Monoid CloudFrontMetadata where
 
 
 -------------------------------------------------------------------------------
-data CloudFrontConfiguration qt = CloudFrontConfiguration
+data CloudFrontConfiguration qt = CloudFrontConfiguration {
```

Comment:
What does the phantom type parameter mean here?

Reply:
The phantom parameter is redundant. 
I think it appeared after copy-pasting configuration type from master library "aws".

Reply:
i don't understand what you mean. This is a type that we define in this library and use it only within the library. I don't see how we're required to have this type parameter. I can't really get this project to even build on a recent compiler or stack LTS. Maybe you could let me know how you're building it locally or perhaps we need to upgrade it to work with stack and a recent LTS.

Reply:
argument is required because  SignQuery class is imported from aws. It has 
an associated type:

````
type ServiceConfiguration request :: * -> * 
```
This class is instantiated half dozen times in aws-cloudfront:

```
instance SignQuery GetInvalidationRequest where
  type ServiceConfiguration GetInvalidationRequest = CloudFrontConfiguration

```

If I remove the phantom ghc complains:

```

src/Aws/CloudFront/Commands/GetInvalidationRequest.hs:33:54: error:
    • Expected kind ‘* -> *’,
        but ‘CloudFrontConfiguration’ has kind ‘*’
    • In the type ‘CloudFrontConfiguration’
      In the type instance declaration for ‘ServiceConfiguration’
      In the instance declaration for ‘SignQuery GetInvalidationRequest’
```

Reply:
I see. Ok, if you just add a brief comment explaining that to the datatype I can go ahead and merge this.

Reply:
@MichaelXavier, thanks. I added comment about qt parameter.